packages/pd-manager-rest-api/__tests__/database/config.ts

```diff
@@ -0,0 +1,22 @@
+import { DataStoreContactEntities } from '@sphereon/ssi-sdk.data-store'
+import { SqliteConnectionOptions } from 'typeorm/driver/sqlite/SqliteConnectionOptions'
+import { DataStoreContactMigrations } from '@sphereon/ssi-sdk.data-store/dist/migrations/generic'
+import { Entities as VeramoDataStoreEntities } from '@veramo/data-store'
+import { migrations as VeramoDataStoreMigrations } from '@veramo/data-store/build/migrations'
+
+const DB_CONNECTION_NAME = 'default'
+const DB_ENCRYPTION_KEY = '29739248cad1bd1a0fc4d9b75cd4d2990de535baf5caadfdf8d8f86664aa830c'
+
+const sqliteConfig: SqliteConnectionOptions = {
+  type: 'sqlite',
+  database: '__tests__/database/test.sqlite',
+  entities: [...DataStoreContactEntities, ...VeramoDataStoreEntities],
```

Comment:
why does this have contact entities? this should get pd entities

Reply:
because this module was a 1:1 copy of the contact manager service and I understood not to work on pd-manager-rest-api anymore

Reply:
fixed